applications/GeoMechanicsApplication/custom_utilities/ublas_utilities.cpp

```diff
@@ -22,4 +22,18 @@ Vector UblasUtilities::CreateVector(const std::initializer_list<double>& rInitia
     std::ranges::copy(rInitializerList, result.begin());
     return result;
 }
+
+Matrix UblasUtilities::CreateMatrix(const std::initializer_list<std::initializer_list<double>>& rInitializerList)
```

Comment:
The code suggests that we receive a list of rows. Perhaps we can rename the function parameter such that this is clearer? For instance:
```suggestion
Matrix UblasUtilities::CreateMatrix(const std::initializer_list<std::initializer_list<double>>& rRows)
```

Reply:
Good suggestion, done